feature/bgp/gracefulrestart/otg_tests/bgp_graceful_restart_test/metadata.textproto

```diff
@@ -0,0 +1,56 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid: "d2e0ad15-fd22-408b-961c-ec016de29925"
+plan_id: "RT-1.4"
+description: "BGP Graceful Restart"
+testbed: TESTBED_DUT_ATE_2LINKS
+platform_exceptions: {
+  platform: {
+    vendor: CISCO
+    hardware_model: "CISCO-8808"
+    hardware_model: "CISCO-8202-32FH-M"
+    hardware_model: "8201"
+    hardware_model: "xrd"
+  }
+  deviations: {
+    ipv4_missing_enabled: true
+  }
+}
+platform_exceptions: {
+  platform: {
+    vendor: NOKIA
+    hardware_model: "7250-ixr10e"
+    hardware_model: "ixr10"
+  }
+  deviations: {
+    use_vendor_native_acl_config: true
```

Comment:
Deviation not required; this was a limitation of IxNetwork (OTG supports GR natively - no need for ACLs)

Reply:
removed